src/spikeinterface/core/numpyextractors.py

```diff
@@ -94,37 +99,63 @@ def get_traces(self, start_frame, end_frame, channel_indices):
 
 
 class NumpySorting(BaseSorting):
+    """
+    In memory sorting object.
+    The internal representation is always done with a long "spike vector".
+
+
+    But we have convinient function to instantiate from other sorting object, from time+labels,
+    from dict of list or from neo.
+
+    Parameters
+    ----------
+    spikes:  numpy.array
+        A numpy vector, the one given by Sorting.to_spike_vector().
+    sampling_frequency: float
+        The sampling frequency in Hz
+    channel_ids: list
+        A list of unit_ids.
+    """
+
     name = "numpy"
 
-    def __init__(self, sampling_frequency, unit_ids=[]):
+    def __init__(self, spikes, sampling_frequency, unit_ids):
```

Comment:
How funny that I neve realized that the intialization was done always done with static method. 

Very elegant that you fit this right into it. As if this is the right datastructure to initialize this object. I like it.